scripts/graph_use_case.py

```diff
@@ -0,0 +1,60 @@
+import os
+
+import torch
+from torch_geometric.data import InMemoryDataset, download_url
+from torch_geometric.transforms import BaseTransform
+
+
+class PlCourtGraphDataset(InMemoryDataset):
+    URL = (
+        "https://huggingface.co/datasets/JuDDGES/pl-court-graph/resolve/main/"
+        "data/pyg_judgement_graph.pt?download=true"
+    )
+
```

Comment:
copy code to tests to ensure that it is working

Reply:
Added, I'll check how it affect CI performance, because it has to download 1.5GB file

Reply:
Whole CI takes 6 min (3 min for unit testing). I think it's acceptable